### ISSUE TITLE
Offer "declare bankruptcy" in Android app

### ISSUE THREAD

**Issue by timabbott** [zulip/zulip-android#326]
If you're like 1000 messages behind on your Zulip history and try to use Zulip on mobile, you end up having to scroll through a ton of messages to get past them to more current things (effectively means you end up needing to catching up on the web).

We should have an option (maybe in the hamburger menu) to "declare bankruptcy" on unread messages and go to the end.  It can work similarly to how it works in the web implementation; see `exports.declare_bankruptcy` in `static/js/unread.js`.

**Comment by saketkumar95** [comment]
@timabbott We can have a icon like this 
![ic_scroll_to_end_fab](https://cloud.githubusercontent.com/assets/21558765/22315222/6f709d76-e38b-11e6-8f3a-2fc22ecf3f2f.png)

on the home screen only. On clicking on it , it will scroll down to current messages. What say?

**Comment by timabbott** [comment]
Well, doing that will mark all the messages as read, which is a fairly disruptive action; I guess maybe we can have it ask in a confirmation modal if there are a large number of messages being skipped over?

**Comment by Sam1301** [comment]
Very interesting and useful feature.. will work on it.

**Comment by yadav-rahul** [comment]
Yes. This feature will be very useful. @Sam1301 I am also interested to be the part of this issue.

**Comment by Sam1301** [comment]
@yadav-rahul cool, we could work together on this issue.

**Comment by yadav-rahul** [comment]
@Sam1301 OK.
